src/components/Link/_link.scss

```diff
@@ -99,4 +99,62 @@
       }
     }
   }
+
+  &--link {
+    border-bottom: 0;
+    font-size: $link-button-label-size;
+    font-weight: 600;
+    letter-spacing: 1px;
+    line-height: $link-label-line-height;
+    text-decoration: underline;
+
+    &:hover {
+      border-bottom: 0;
+    }
+  }
+
+  &--snwHeaderLink {
```

Comment:
This looks like a duplicate of line 128. The styles are slightly different under 128 though so maybe consolidate them and remove the duplicate styles. Let me know if this was intentional and I'll review the PR again